chart/tests/helm_template_generator.py

```diff
@@ -29,7 +29,7 @@
 
 api_client = ApiClient()
 
-BASE_URL_SPEC = "https://raw.githubusercontent.com/instrumenta/kubernetes-json-schema/master/v1.12.9"
```

Comment:
version 1.12.9 did not have `ingress-networking-v1beta1.json` so we could not validate the template output. Version 1.14.0 is the first to include  `ingress-networkin-v1beta1.json`

Reply:
In that case, we also need to update README.md.
https://github.com/apache/airflow/tree/master/chart#prerequisites
This works for me as these versions are no longer supported by GCP, Azure, AWS, so we can raise the version requirements.
```
$ gcloud container get-server-config --zone us-central1-a
Fetching server config for us-central1-a
channels:
- channel: RAPID
  defaultVersion: 1.18.10-gke.1500
  validVersions:
  - 1.18.10-gke.2101
  - 1.18.10-gke.1500
- channel: REGULAR
  defaultVersion: 1.17.12-gke.1504
  validVersions:
  - 1.17.13-gke.1400
  - 1.17.12-gke.1504
- channel: STABLE
  defaultVersion: 1.16.13-gke.404
  validVersions:
  - 1.16.15-gke.4300
  - 1.16.13-gke.404
  - 1.16.13-gke.401
  - 1.15.12-gke.6001
  - 1.15.12-gke.20
```
https://docs.microsoft.com/pl-pl/azure/aks/supported-kubernetes-versions#aks-kubernetes-release-calendar
https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html

Reply:
Ok, I've raised the prerequisite to 1.14, but I 'd like to note that was already a prerequisite (if you wanted to use ingresses). But now we validate helm chart outputs (ingresses or not) to 1.14 so it's a good idea to require 1.14

Reply:
We have a new test framework, which as you can see, came in handy and detected a problem that the previous tool did not detect.
See: 
https://github.com/apache/airflow/issues/11657
https://github.com/apache/airflow/pull/12003
I am glad that we were able to detect this problem with this framework and can update the documentation.